### ISSUE TITLE
Change case on return value & format code for readability

### ISSUE THREAD

**Issue by cliffhall** [MaxflowO2/ERC2981#2]
https://github.com/MaxflowO2/ERC2981/blob/010744cac3e6c5214a184733faa59cca61884daa/contracts/ERC2981.sol#L49

Readability is important in reference and framework code. Formatting it with that in mind (as we did in the ERC), will be appreciated by those who evaluate it when considering adoption.

```
function royaltyInfo(uint256 _tokenId, uint256 _salePrice) 
external 
view 
override(IERC2981) 
returns (address Receiver, uint256 royaltyAmount) { }
```

Also, return values should begin with lower case `receiver` not `Receiver`.

**Comment by MaxflowO2** [comment]
Yeah long night when I wrote this, but out of my silly season for a bit will update accordingly
